server/docs/patterns.md

```diff
@@ -37,6 +39,16 @@ CREATE TABLE `sample` (
 );
 ```
 
+### UUIDs
+
+Use `binary` (or `varbinary`) data type for UUIDs. [MySQL 8 has good support for UUIDs](https://dev.mysql.com/blog-archive/mysql-8-0-uuid-support/) with `UUID_TO_BIN` and `BIN_TO_UUID` functions. If needed, add a virtual table to display UUID as string. [Backend sync where discussed](https://us-65885.app.gong.io/call?id=5477893933055484926&highlights=%5B%7B%22type%22%3A%22SHARE%22%2C%22from%22%3A440%2C%22to%22%3A612%7D%5D).
+
+Benefits of binary UUIDs include:
+- Smaller storage size
+- Faster indexing/lookup
```

Comment:
:+1: that's a great proposal, just want to add a heads-up that in some cases (Windows vs Apple MDM profiles at least, might be the only place?) we add a one-letter prefix to uuids due to historical reasons of how those profiles were supported (started Apple only, Windows in distinct tables and we need to know in some places in what table we need to lookup the uuid), so those cannot be changed to binary format (they are varchar 37 instead of 36 anyway).